src/yang/example-compa-diffserv.yang

```diff
@@ -32,60 +32,73 @@ module example-compa-diffserv {
      companyA diffserv specification extension.";
   revision YYYY-MM-DD {
     description
-      "Initial revision for diffserv actions on network packets";
+      "Initial revision for diffserv actions on network packets.";
     reference
       "RFC XXXX: A YANG Data Model for Quality of Service (QoS).";
   }
 
   identity default-policy-type {
     base qos-types:policy-type;
     description
-      "This defines default policy-type";
+      "This defines default policy-type.";
   }
 
   identity qos-group {
     base qos-types:filter-type;
     description
-      "qos-group filter-type";
-  }
-
-  identity drop {
-    base qos-types:action-type;
-    description
-      "drop action-type";
+      "Filter-type to match on  packet metadata.";
```

Comment:
First of all, the name of the identity qos-group is confusing to me. What is a qos-group? The term has not been used in the draft anywhere. Can the name reflect what the identity really is. 

I also do not understand this description. Is this an identity that defines filtering based on matches within a packet metadata? If so, it would help to reword this to:

"A filter type that matches on packet metadata."

But the larger question is, what is packet metadata? This is the first time I am hearing about packet metadata. Are these fields within a packet? If so, can we use that in the description.

Reply:
this is cisco specific.. so we may not look much into it?

Reply:
Even if it a Cisco specific example, it needs to be clear. I would rather that we spend the time to clean it up and make it clear what the example is trying to demonstrate.

Reply:
Mahesh, it is same as traffic-group in our model.. traffic-group is converted into an integer internally.. QoS-group is an integer from config. Also, traffic-group is mainly used for queuing, QoS-group can be used for other things also since 256 values are possible while traffic-groups are typically 8.

Reply:
I have clarified further. Please check.

Reply:
Thanks. Can we do something similar for the description of qos-group on line 58?